streams/shared/src/main/scala/zio/stream/experimental/ZStream.scala

```diff
@@ -295,11 +295,38 @@ class ZStream[-R, +E, +A](val channel: ZChannel[R, Any, Any, Any, E, Chunk[A], A
 
   /**
    * Allows a faster producer to progress independently of a slower consumer by buffering
-   * up to `capacity` chunks in a queue.
+   * up to `capacity` elements in a queue.
    *
    * @note Prefer capacities that are powers of 2 for better performance.
```

Comment:
Can you add another note that this combinator destroys the chunking structure and it is recommended to add a call to `chunkN` afterwards?

Reply:
Done